lib/v2/taptap-intl/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (router) => {
+    router.get('/changelog/:id/:lang?', require('./changelog'));
+    router.get('/review/:id/:order?/:lang?', require('./review'));
+};
```

Comment:
Consider reusing the `taptap` namespace and adding `/intl` before new routes instead of creating new namespace `taptap-intl`